.github/workflows/build-and-test.yml

```diff
@@ -202,9 +202,16 @@ jobs:
 
       - name: ${{ github.event_name == 'push' && 'Package and push' || 'Package' }} helm charts
         run: |
-          if [[ "${{ github.event_name }}" == "push" && "${{ github.ref }}" == "refs/heads/main" ]]; then
-            echo "Packaging and pushing Helm charts..."
-            make helm-push
+          if [[ "${{ github.event_name }}" == "push" ]]; then
```

Comment:
The code now pushes charts for any branch trigger. Is it our intention to allow per-commit publishing for all branches (including feature branches), or should we restrict the SHA-push to main and release-v* specifically?

Reply:
Ack. Noted that the workflow-level trigger is already restricted to main and release-v*